pkg/app/piped/cloudprovider/lambda/client.go

```diff
@@ -195,12 +203,27 @@ func (c *client) PublishFunction(ctx context.Context, fm FunctionManifest) (vers
 }
 
 // RoutingTrafficConfig presents a map of primary and secondary version traffic for lambda function alias.
-type RoutingTrafficConfig map[string]VersionTraffic
+type RoutingTrafficConfig map[TrafficConfigKeyName]VersionTraffic
+
+func (c *RoutingTrafficConfig) Encode() (string, bool) {
+	out, err := json.Marshal(c)
+	if err != nil {
+		return "", false
+	}
+	return string(out), true
+}
+
+func (c *RoutingTrafficConfig) Decode(data []byte) bool {
+	if err := json.Unmarshal(data, c); err != nil {
+		return false
```

Comment:
I think it would be better to return "error" instead of "bool", example for logging.
Same with the "Encode" function.

Reply:
🙆‍♀️ addressed on https://github.com/pipe-cd/pipe/pull/1462/commits/fc8fffb205c65305d7bb558c77c9c31d73b9464a